### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by CebiConRepo** [2sic/2sxc#2503] [General] [answered]
Hi there,

Unfortunately, a rebuild on the "Develop" branch does not work for me. As I am currently trying to build a project (website for a hotel) with Oqtane, I would like to use the "2sxc" module. Hence my question (bug because of no edit) when can a new release be expected?

Thanks for the information

**Comment by iJungleboy** [comment, marked as answer]
Within the next 48 hours 😉

We're working hard on ensuring that both 2sxc is up to date, and that the content-templates work with the bootstrap 5 changes of Oqtane. Almost there...

**Reply by CebiConRepo** [reply]
great thanks too
